doc/source/cluster/running-applications/job-submission/ray-client.rst

```diff
@@ -26,13 +26,24 @@ By changing ``ray.init()`` to ``ray.init("ray://<head_node_host>:<port>")``, you
    do_work.remote(2)
    #....
 
+When to use Ray Client
+----------------------
+
+Ray Client should be used when you want to connect a script or an interactive shell session to a **remote** cluster.
+
+* Use ``ray.init("ray://<head_node_host>:10001")`` (Ray Client) if you've set up a remote cluster at ``<head_node_host>`` and you want to do interactive work. This will connect your local script or shell to the cluster. See the section on :ref:`using Ray Client<how-do-you-use-the-ray-client>` for more details on setting up your cluster.
+* Use ``ray.init("localhost:<port>")`` (non-client connection, local address) if you're developing locally or on the head node of your cluster and you have already started the cluster (i.e. ``ray start --head`` has already been run)
```

Comment:
Actually, as of Ray 2.0.0, `ray.init()` is now the way to go if you've run `ray start` already.
You only need to specify an address if there are multiple Ray instances running locally.